share/functions/__fish_print_function_prototypes.fish

```diff
@@ -1,5 +1,5 @@
 
 function __fish_print_function_prototypes -d "Prints the names of all function prototypes found in the headers in the current directory"
-    cat *.h* | sed -n "s/^\(.*[^[a-zA-Z_0-9]\|\)\([a-zA-Z_][a-zA-Z_0-9]*\) *(.*);.*\$/\2/p"
+    sed -n "s/^\(.*[^[a-zA-Z_0-9]\|\)\([a-zA-Z_][a-zA-Z_0-9]*\) *(.*);.*\$/\2/p" *.h*
```

Comment:
This glob will error if unmatched (i.e. when you're in the wrong directory). It needs to be assigned to a variable.